net-mgmt/pfSense-pkg-softflowd/pkg-descr

```diff
@@ -5,7 +5,7 @@ by listening on a network interface or by reading a
 packet capture file. These flows may be reported via
 NetFlow to a collecting host or summarised within
 softflowd itself.<br/> Softflowd supports Netflow
-versions 1, 5 and 9 and is fully IPv6-capable - it can
+versions 1, 5, 9 and 10 (IPFIX) and is fully IPv6-capable - it can
 track IPv6 flows and send export datagrams via IPv6. It
 also supports export to multicast groups, allowing for
-redundant flow collectors.
+redundant flow collectors.
```

Comment:
Please keep newline `\n` at the end of pkg-descr, otherwise portlint will complain

Reply:
I've added the \n at the end of the pkg-descr as per your suggestion. However, oddly enough, when I run portlint (2.17.15), it complains (with a warning) of a "unnecessary blank lines at the last part".

Reply:
Yeah, you can't have a bank line in the end of file. It's mostly an editor setting usually called "newline at EOF". Please remove the blank line